tests/settings.py

```diff
@@ -94,6 +95,15 @@
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
 ]
 
+# middleware list for newer djangos
+MIDDLEWARE = MIDDLEWARE_CLASSES
+
+# session auth middleware is not needed since django 1.10
+# see https://docs.djangoproject.com/en/3.0/releases/2.0/#miscellaneous
+if django.VERSION >= (1, 10):
```

Comment:
Is 1.10 a supported version?
in tox file I see only 1.11 LTS

```
deps =
    django-111: Django>=1.11,<1.12	    django-111: Django>=1.11,<1.12
    django-20: Django>=2.0,<2.1	    django-20: Django>=2.0,<2.1
```

Reply:
Yep, it's not supported by this library. So what you suggest is to remove session auth middleware altogether?

Reply:
nuked